projects/fusion-e2e/tests/components/alert/alert-component.ts

```diff
@@ -1,11 +1,13 @@
 import {Page} from '@playwright/test';
 import {AlertTestIdModifiers} from '@ironsource/fusion-ui/entities';
 import {getTestId, getTestIdSelector} from '../../global/utils';
+import {BaseComponent} from '../base-component';
 
-export class AlertComponent {
+export class AlertComponent extends BaseComponent {
     readonly page: Page;
 
-    constructor(page: Page) {
+    constructor(page: Page, selector: string) {
+        super(page, selector);
```

Comment:
You are still using playwright methods directly instead of using the wrapped methods under base element page. The whole point of this class is that you will not usr page.click / waitForSelector directly

Reply:
Done